product_docs/docs/eprs/7/eprs_rel_notes/eprs_rel_notes_7.8.0.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Replication Server 7.8.0 release notes"
+navTitle: Version 7.8.0
+---
+
+Released: 17 May 2024
+
+New features, enhancements, bug fixes, and other changes in Replication Server 7.8.0 include the following:
+
+| Type        | Description                                                                                                                                                                                                                                 | Addresses                  |
+|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------|
+| Enhancement | Added the ability to retry connecting to a target/subscription database if the connection is lost during a snapshot replication.                                                                                                            |                            |
+| Enhancement | Added the `skipColDefaultClause` subscription server configuration option to skip the column DEFAULT value for the subscription database.                                                                                                   | #100987/32874              |
+| Enhancement | EDB Postgres Replication Server now supports installations for SLES15 SP5 packages.                                                                                                                                                         |                            |
```

Comment:
I suggest updating the description to the following:

> EDB Postgres Replication Server now provides packages for and supports installations on SLES15 SP5

@anwar-sahib thoughts?

Reply:
@mlewandowski-edb your description is more explicit, however I feel both the text essentially conveys the same thing that one can now install EPRS on SLES15 SP5. I am fine with any text.